types/mini-css-extract-plugin/index.d.ts

```diff
@@ -44,6 +44,12 @@ declare namespace MiniCssExtractPlugin {
          * @default true
          */
         esModule?: boolean;
+        /**
+         * This option allows loading asynchronous chunks with a custom link type, such as <link type="text/css" ...>.
+         * `false` disables the link `type` attribute
+         * @default `text/css`
+         */
+        linkType?: string | boolean;
```

Comment:
``` suggestion
linkType?: false | 'text/css';
```

Reply:
AFAIK it can be 'true' and the code expect true, that's why I went with `boolean`:
https://github.com/webpack-contrib/mini-css-extract-plugin/blob/60c3eef936245183451d63f1e080ce990553078d/src/index.js#L52

Reply:
OK, then it should be

```suggestion
        linkType?: boolean | 'text/css';
```

Reply:
yup, you're correct here, thx!